service/spawn.go

```diff
@@ -310,7 +311,13 @@ func (uis *UIServer) modifySpawnHost(w http.ResponseWriter, r *http.Request) {
 
 	case HostExpirationExtension:
 		if updateParams.Expiration.IsZero() { // set expiration to never expire
-			if err := route.CheckUnexpirableHostLimitExceeded(u.Id, uis.Settings.UnexpirableHostsPerUser); err != nil {
+			settings, err := evergreen.GetConfig()
```

Comment:
I would get this from uis

Reply:
Without these changes, the spawn host limit was [fetched from the db](https://github.com/evergreen-ci/evergreen/blob/6a0d0c2c491523dbdf0ed4f16205ef7db20b3367/service/spawn.go#L59) and the unexpirable host limit was pulled from the env and I thought they should be consistent. Do you think they should both be pulled from the env?

Reply:
We had an in person conversation about this, where I pushed for getting it from the db settings:
In the case of a problem, we would change this number to 0, and I think that having to restart the app servers to put this change into effect shouldn't be required since that can lead to problems.